[ERROR] 20:07:38:107 CLIENT/(5550:5550): CONNECT_ERROR#16 (src/client.c:client_send_msg:93) -- Connection refused

[TRACE] 20:09:36:593 CLIENT/(5651:5651): Iniciado correctamente.
